ui/analyse/src/patch.ts

```diff
@@ -1,3 +1,3 @@
-import { attributesModule, classModule, init } from 'snabbdom';
+import { attributesModule, classModule, eventListenersModule, init } from 'snabbdom';
 
-export default init([classModule, attributesModule]);
+export default init([classModule, attributesModule, eventListenersModule]);
```

Comment:
please don't pull the module in ui/analyse just to add a click handler. You'll find example of click handlers around that don't use it.

Reply:
ok ... I thought that much :)

I will have a look

Reply:
done